addon/components/frost-list.js

```diff
@@ -155,6 +172,11 @@ export default Component.extend({
       })
     }
 
+    const itemTypeKey = this.get('itemTypeKey')
+    if (!itemKey && itemTypeKey) {
+      Logger.warn('If itemTypeKey is defined, then itemKey needs to be defined as well')
```

Comment:
Is there any relationship between itemKey and itemTypeKey?

Reply:
No relationship, no. `itemKey` is the name of the key that uniquely identifies each item, and `itemTypeKey` is the name of the key that indicates the item's type. I will double check why we need this warning and remove it if it's unnecessary.